worker-languagedetection-container/testcases/automated/CAF_4001/kinyarwanda.yaml.testcase

```diff
@@ -37,43 +37,43 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "rw"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "73"
```

Comment:
Significant change here, cannot identify kinyarwanda anymore. If it is not supported then remove this testcase and make sure that kinyarwanda is not referred to as a supported language.